### PR TITLE
Feature/issue 80 style questionnaire

### PR DESCRIPTION
**Pull request by bensinjin**

### CODE REVIEW

File: src/selectors/questionnaire.ts [hunk 1]
```diff
@@ -26,19 +28,27 @@ export const denormalizeQuestions = (locale: Locale, modelStore: model.Store): Q
...
-            id,
-            text: selectLocalizedText(locale, text),
-            answers: selectAnswersForQuestion(locale, id, answers),
-        };
+        const question: model.Question = questions[key];
+        return question.explanation ?
+            {
+                id: question.id,
+                text: selectLocalizedText(locale, question.text),
+                explanation: selectLocalizedText(locale, question.explanation),
```
[the remark this answers is not in the document]
Comment (bensinjin) [reply src/selectors/questionnaire.ts]: Fixed

File: src/components/questionnaire/answer.tsx [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
+const RadioAnswer: React.StatelessComponent<Props & Actions> = (props: Props & Actions): JSX.Element => (
+    renderAnswer(props, <Radio selected={props.answer.isSelected} />)
 );
 
-const SelectedAnswer: React.StatelessComponent<Props & Actions> = (props: Props & Actions): JSX.Element => (
-    <View style={{ padding: 3 }}>
-        <Button rounded block onPress={(): SelectAnswerAction => props.selectAnswer(props.answer.id)}>
+const renderAnswer: React.StatelessComponent = (props: Props & Actions, component: JSX.Element): JSX.Element => (
+    <ListItem button noBorder onPress={(): SelectAnswerAction => props.selectAnswer(props.answer.id)}>
+        <Body>
```
[the remark this answers is not in the document]
Comment (bensinjin) [reply src/components/questionnaire/answer.tsx]: Fixed the inability to click the checkbox/radio.  Are these questions the ones for the MVP? I think avoiding the issue of answers affecting questions is preferable for the MVP but in this case could probably be addressed by having one question

File: src/components/questionnaire/questionnaire.tsx [hunk 3]
```diff
@@ -11,16 +14,13 @@ export interface Props {
...
-            padding: 10,
-        }}>
-            {props.questionnaire.map((question: selector.Question) => (
-                <Question key={question.id} question={question} selectAnswer={props.selectAnswer} />
+    <Container>
+        <Content padder>
+            <Text style={applicationStyles.pageTitle}><Trans>Personalize My Plan</Trans></Text>
+            <Text style={questionnaireStyles.introText}>Immigration can be overwhelming but we are here to help!</Text>
+            {props.questionnaire.map((question: selector.Question, index: number) => (
+                <Question key={question.id} question={question} questionNumber={index + 1} selectAnswer={props.selectAnswer} />
```
[the remark this answers is not in the document]
Comment (bensinjin) [reply src/components/questionnaire/questionnaire.tsx]: I've moved the calculation of the page number into `denormalizedQuestions()`. I think it's probably up for debate how or where we get this value from but this should be ok for now.

File: src/components/questionnaire/answer.tsx [hunk 4]
```diff
@@ -11,25 +11,32 @@ export interface Props {
...
-    </ListItem>
-);
+const getComponentForAnswerType = (props: Props, answerType: AnswerType, onPress: () => void): JSX.Element => {
+    switch (answerType) {
+        case AnswerType.CheckboxAnswer:
+            return <CheckBox checked={props.answer.isSelected} onPress={onPress} />;
+        case AnswerType.RadioAnswer:
+        default:
+            return <Radio selected={props.answer.isSelected} onPress={onPress} />;
+    }
```
Comment (rasmus-storjohann-PG) [inline src/components/questionnaire/answer.tsx]: This looks like a switch statement but it's actually equivalent to an if-else statement. The code would be more readable if you just wrote if-else. Very minor point, not worth holding up the PR.

File: src/components/questionnaire/questionnaire.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (rasmus-storjohann-PG) [reply src/components/questionnaire/questionnaire.tsx]: Yes, we will need to figure this out. Probably post MVP unless questions appear in confusing order, then we may need to fix this sooner.

**Pull request merged**